label/label.go

```diff
@@ -63,6 +66,12 @@ type GormLabelRepository struct {
 	db *gorm.DB
 }
 
+// TableName overrides the table name settings in Gorm to force a specific table name
+// in the database.
+func (m *GormLabelRepository) TableName() string {
```

Comment:
This could be a constant.

Reply:
right, no need of `TableName` in repo level. Done in https://github.com/fabric8-services/fabric8-wit/pull/1627/commits/1213f9fd2557e3a992c11736ab24e7be3af1e0fe